screen2.0/src/app/applets/differential-gene-expression/page.tsx

```diff
@@ -102,591 +111,442 @@ export default function DifferentialGeneExpression() {
+                            expID: row.expID,
+                          },
+                          ct2: title.ct2,
+                        })
+                        // if (rowHighlight) setRowHighlight([row, rowHighlight[1]])
+                        // else setRowHighlight([row])
+                      }}
+                      sortDescending={true}
+                      searchable={true}
+                    />
+                      </AccordionDetails>
+                    </Accordion>
+                    
+                  </Box>
+                  <Divider sx={{ mb: 2, mt: 2 }} />
+                  <Box sx={{  }}>
+                    <Accordion defaultExpanded={true}>
+                    <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                        <Typography variant="h5">Cell Type 2</Typography>
+                      </AccordionSummary>
+                      <AccordionDetails>
+                        <DataTable
+                      // tableTitle="Cell type 2"
+                      // highlighted={rowHighlight ? rowHighlight[1] : false}
+                      page={1}
+                      rows={cellTypes["cellTypeInfoArr"]}
+                      columns={[
+                        { header: "Cell Type", value: (row: any) => row.biosample_summary },
+                        { header: "Experiment", value: (row: any) => row.expID, render: (row: any) => createLink("https://encodeproject.org/experiments/", row.expID)},
+                        { header: "Tissue", value: (row: any) => row.tissue },
+                      ]}
+                      onRowClick={(row: any) => {
+                        setct2(row.value)
+                        setTitle({
+                          ct1: title.ct1,
+                          ct2: {
+                            name: row.name,
+                            expID: row.expID,
+                          },
+                        })
+                        // if (rowHighlight) setRowHighlight([rowHighlight[0], row])
+                        // else setRowHighlight([row])
+                      }}
+                      sortDescending={true}
+                      searchable={true}
+                    />
+                      </AccordionDetails>
+                    </Accordion>
+                    
+                  </Box>
+                  <Divider sx={{ mb: 2, mt: 2 }} />
+                  <Grid2 xs={12} md={12} lg={12} ml={1} mr={1}>
+                  <Typography>Filters</Typography>
+                  <Divider sx={{ mb: 2, mt: 2 }} />
+                  <FormGroup>
+                    <TogglePlot label="cCREs" toggle={toggleccres} setToggle={setTogglecCREs} />
+                    <TogglePlot label="log2 fold change" toggle={toggleFC} setToggle={setToggleFC} />
+                    <TogglePlot label="genes" toggle={toggleGenes} setToggle={setToggleGenes} />
+                  </FormGroup>
+                  <Divider sx={{ mb: 2, mt: 2 }} />
```

Comment:
Can this sections of filters be named too?

Reply:
not sure what to name this, can talk about it during our scrum